redis_cache/rediscache.py

```diff
@@ -211,8 +214,16 @@ def get(self, key):
             if value is None:  # expired key
                 if not key in self:  # If key does not exist at all, it is a straight miss.
                     raise CacheMissException
-
-                self.connection.srem(self.get_set_name(), key)
+                try:
+                    self.connection.srem(self.get_set_name(), key)
+                except ReadOnlyError:
+                    # Connect to the failover write node
+                    self.connection = RedisConnect(host=self.failoverhost,
```

Comment:
since the default for failoverhost is `None` could it be possible that we set the host to none? Just wondering if we would want to check this before using it and maybe output a separate error if we hit the ReadOnlyError and there is no failoverhost. I dunno if that's a likely scenario, looking for your thoughts on that.

Reply:
Yeah, you are right.

Reply:
PR updated.